src/components/Alert/Alert.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { ReactNode } from 'react';
+
+import { IconCheck, IconAlertCircle } from '../Icons';
+
+import styles from './Alert.module.scss';
+import classNames from 'classnames';
+
+type AlertVariant = 'success' | 'error' | 'warning' | 'info';
+
+export interface AlertProps {
+  className?: string;
+  variant?: AlertVariant;
+  children?: ReactNode;
+}
+
+export const Alert = ({ className, variant, children }: AlertProps) => {
+  return (
+    <div className={classNames(styles.Container, className)} data-variant={variant}>
+      <Icon variant={variant} />
+      <div>{children}</div>
+    </div>
+  );
+};
+
+interface IconProps {
+  variant: AlertVariant;
+}
+
+const Icon = ({ variant }: IconProps) => {
+  return variant === 'success' ? <IconCheck size={16} /> : <IconAlertCircle size={16} />;
```

Comment:
Do we use the same icon for `info | warning | error`? This might be helpful.

![image](https://user-images.githubusercontent.com/101390821/196133353-43380a86-2b20-4a21-92d6-b4b1727613dd.png)

Reply:
Same icon, different colours.

<img width="147" alt="Screen Shot 2022-10-17 at 9 58 02 PM" src="https://user-images.githubusercontent.com/12437916/196135115-c7d968b6-92ab-410f-86a0-2dabaff033fd.png">

Reply:
Yes, I think we need to use another icon for `warning & error` variant